### PR TITLE
fix(style): use checkmark's default size instead of customizing

### LINKED ISSUE #222: checkmark size is incorrect in select and combobox
### Please verify the version of auro-menu you have installed

_No response_

### Please describe the bug

1. In https://auro.alaskaair.com/components/auro/select/ example, 
2. Select a value on `Auro-select` and reopen `Auro-select` 
3. You can see a giant checkmark 
image

### Reproducing the error

This issue is reproducible on the Auro docsite

### Expected behavior

_No response_

### What browsers are you seeing the problem on?

_No response_

### Additional context

_No response_

### Exit criteria

_No response_

Related to issue https://github.com/AlaskaAirlines/auro-alert/issues/71

### PR DESCRIPTION
**Pull request by sun-mota**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Resolves:** #222

## Summary:

This is to fix a bug which was showing a big checkmark on selected menu option.
Instead of customizing the checkmark icon size, this fix will just leave the icon as default size. 

## Type of change:

Please delete options that are not relevant.

- [ ] New capability
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

## Summary by Sourcery

Bug Fixes:
- Fix the oversized checkmark icon issue by removing custom size settings and using the default size.

### CODE REVIEW

**Comment by sun-mota** [comment]
looks good to me, but cannot add an approval as I contributed in the code. ;)

File: src/style-menuoption.scss [hunk 1]
```diff
@@ -33,8 +33,7 @@
   }
 
   [auro-icon] {
-    height: var(--ds-size-300, $ds-size-300);
-    width: var(--ds-size-300, $ds-size-300);
+    width: $ds-size-300;
```
Comment (jason-capsule42) [inline src/style-menuoption.scss]:
```suggestion
   width: var(--ds-size-300, $ds-size-300);
```

Comment (jason-capsule42) [reply src/style-menuoption.scss]: We can't remove tokens from here or we break theme support.

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]

File: src/auro-menuoption.js [hunk 2]
```diff
@@ -117,7 +117,7 @@ export class AuroMenuOption extends LitElement {
       ${this.selected && !this.nocheckmark ? html`
         <${this.iconTag}
           ondark
-          customSize
+          customsize
```
Comment (jordanjones243) [inline src/auro-menuoption.js]: This attribute needs to be removed, as the `customSize` attribute is no longer supported.

**Review by jordanjones243 — CHANGES_REQUESTED** [review]

**Review by jordanjones243 — DISMISSED** [review]

File: src/style-menuoption.scss [hunk 3]
```diff
@@ -33,8 +33,7 @@
   }
 
   [auro-icon] {
-    height: var(--ds-size-300, $ds-size-300);
-    width: var(--ds-size-300, $ds-size-300);
+    --ds-auro-icon-size: var(--ds-size-300,$ds-size-300);
```
Comment (jason-capsule42) [inline src/style-menuoption.scss]:
```suggestion
    --ds-auro-icon-size: var(--ds-size-300, $ds-size-300);
```

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]
For some reason the docs are not building correctly. Hold off on releasing that until we assess why that is.

**Review by jason-capsule42 — APPROVED** [review]

**Pull request merged**